specification/oracle/Oracle.Database.Management/tspconfig.yaml

```diff
@@ -52,6 +52,7 @@ options:
     clear-output-folder: true
     flavor: azure
     emitter-output-dir: "{output-dir}/{service-dir}/{namespace}"
+    service-dir: "sdk/oracle"
```

Comment:
does this work? I tested that we can just update emitter-output-dir with this value, seems service-dir is not in use

Reply:
Yes, it works

Reply:
ok, maybe not working in new emitter anymore.